rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -234,42 +358,84 @@ s32 cellCameraGetType(s32 dev_num, vm::ptr<s32> type)
 s32 cellCameraIsAvailable(s32 dev_num)
 {
 	cellCamera.todo("cellCameraIsAvailable(dev_num=%d)", dev_num);
-	return CELL_OK;
+
+	const auto g_camera = fxm::get<camera_thread>();
+
+	if (!g_camera || !check_dev_num(dev_num))
+	{
+		return false;
+	}
+
+	if (g_cfg.io.camera_type == fake_camera_type::uvc1_1)
```

Comment:
what about other cam types?

Reply:
this shouldn't be here, thanks for the heads up